### PR TITLE
Inna kushchak

### PR DESCRIPTION
**Pull request by InnaKushchak**

### CODE REVIEW

**Comment by evanmaiqa** [comment]
Actually between tests we need an empty line. Intellij idea can help with this, just press Ctrl + Alt + L to trigger reformatting of the code.

**Comment by InnaKushchak** [comment]
got it thanks

File: src/test/java/InnaKushchakTest.java [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+        WebElement sign = getDriver().findElement(By.xpath("//span[@class='woocommerce-Price-currencySymbol']"));
+        Assert.assertEquals(sign.getText(), "$");
+    }
+    @Test
+    public void testOffOnAllProducts() {
+        String message = getDriver().findElement(By.xpath("//h3[text()='25% OFF On all products']")).getText();
         Assert.assertEquals(message, "25% OFF On all products");
-
-   
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:22]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 2]
```diff
@@ -1,42 +1,48 @@
...
-
     }
     @Test
-
-    public void OFFOnAllProducts_TC_003()  {
-        String message= driver.findElement(By.xpath("//h3[text()='25% OFF On all products']")).getText();
+    public void  testValidate$SignIsPresent() {
+        WebElement sign = getDriver().findElement(By.xpath("//span[@class='woocommerce-Price-currencySymbol']"));
+        Assert.assertEquals(sign.getText(), "$");
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:17]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 3]
```diff
@@ -1,42 +1,48 @@
...
+    @Test
+    public void testVerifyingLogInButton() {
+        getDriver().findElement(By.xpath("//a[@href='https://askomdch.com/account/']")).click();
+        Assert.assertEquals(getDriver().getCurrentUrl(), "https://askomdch.com/account/");
+    }
+    @Test
+    public void testVerifyingShopNowButton() {
+        getDriver().findElement(By.xpath("//a[text()='Shop Now']")).click();
+        Assert.assertEquals(getDriver().getCurrentUrl(), "https://askomdch.com/store");
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:32]: }

@Test

File: src/test/java/runner/BaseTest.java [hunk 4]
```diff
@@ -24,7 +24,6 @@ protected void beforeMethod() {
     protected void afterMethod() {
         driver.quit();
     }
```
Comment (evanmaiqa) [inline src/test/java/runner/BaseTest.java:26]: }

protected

File: src/test/java/InnaKushchakTest.java [hunk 5]
```diff
@@ -1,42 +1,48 @@
...
-    }
+public class InnaKushchakTest extends BaseTest {
     @Test
-    public void VerifySaleIcon_TC_001()  {
-        WebElement icon = driver.findElement(By.xpath("//span[@class='onsale']"));
+    public void testVerifySaleIcon() {
+        WebElement icon = getDriver().findElement(By.xpath("//span[@class='onsale']"));
         Assert.assertEquals(icon.getText(), "Sale!");
-
     }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:12]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 6]
```diff
@@ -1,42 +1,48 @@
...
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(2));
-        driver.manage().window().setSize(new Dimension(1920, 1080));
-        driver.get("https://askomdch.com/");
-    }
-
-    @AfterMethod
-    public void afterMethod() {
-        driver.quit();
-    }
+public class InnaKushchakTest extends BaseTest {
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:7]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 7]
```diff
@@ -1,42 +1,48 @@
...
+        String message = getDriver().findElement(By.xpath("//h3[text()='25% OFF On all products']")).getText();
         Assert.assertEquals(message, "25% OFF On all products");
-
-   
+    }
+    @Test
+    public void testVerifyingLogInButton() {
+        getDriver().findElement(By.xpath("//a[@href='https://askomdch.com/account/']")).click();
+        Assert.assertEquals(getDriver().getCurrentUrl(), "https://askomdch.com/account/");
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:27]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 8]
```diff
@@ -1,42 +1,48 @@
...
+    @Test
+    public void testVerifyingShopNowButton() {
+        getDriver().findElement(By.xpath("//a[text()='Shop Now']")).click();
+        Assert.assertEquals(getDriver().getCurrentUrl(), "https://askomdch.com/store");
+    }
+    @Test
+    public void testVerify5ProductsDisplayed() {
+        Assert.assertEquals(getDriver().findElement(By.cssSelector("ul.products.columns-5")).findElements(By.tagName("li")).size(), 5);
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:36]: }

@Test

File: src/test/java/InnaKushchakTest.java [hunk 9]
```diff
@@ -1,42 +1,48 @@
...
+    @Test
+    public void testManSearchButtonColorVerification() {
+        getDriver().findElement(By.cssSelector("li[id='menu-item-1228']")).click();
+        Assert.assertEquals(getDriver().findElement(By.xpath("//button[@value='Search']")).getCssValue("background-color"), "rgba(49, 151, 214, 1)");
+    }
```
Comment (evanmaiqa) [inline src/test/java/InnaKushchakTest.java:41]: }

@Test

**Review by evanmaiqa — COMMENTED** [review]
Please add empty lines

**Review by evanmaiqa — APPROVED** [review]

**Pull request merged**